### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by HarshMN2345**
i  made it easy to understand

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
 # Welcome to DocsGPT Contributing Guidelines
 
-Thank you for choosing this project to contribute to. We are all very grateful!
+Thank you for choosing to contribute to our project! We greatly appreciate your help!
```
Comment (dartpain) [inline CONTRIBUTING.md:3]: Instead of our project, just say DocsGPT

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,27 +1,27 @@
...
 
 ### [🎉 Join the Hacktoberfest with DocsGPT and Earn a Free T-shirt! 🎉](https://github.com/arc53/DocsGPT/blob/main/HACKTOBERFEST.md)
 
 # We accept different types of contributions
 
-📣 Discussions - where you can start a new topic or answer some questions
+📣 Discussions - Engage in conversations, start new topics, or help answer questions.
 
-🐞 Issues - This is how we track tasks, sometimes it is bugs that need fixing, and sometimes it is new features
+🐞 Issues - This is where we keep track of tasks. It could be bug fixes or suggestions for new features.
```
Comment (dartpain) [inline CONTRIBUTING.md:11]: bugs, fixes or suggestions ....

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,27 +1,27 @@
...
+🛠️ Pull requests - Suggest changes to our repository, either by working on existing issues or adding new features.
 
-📚 Wiki - where we have our documentation
+📚 Wiki - This is where our documentation resides.
 
 
 ## 🐞 Issues and Pull requests
 
-We value contributions to our issues in the form of discussion or suggestions. We recommend that you check out existing issues and our [roadmap](https://github.com/orgs/arc53/projects/2).
+We highly value contributions in the form of discussions or suggestions. We recommend taking a look at existing issues and our [roadmap](https://github.com/orgs/arc53/projects/2).
```
Comment (dartpain) [inline CONTRIBUTING.md:20]: Maybe remove the word highly. We do value contributions highly, but it just sounds better without the word

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -32,10 +32,10 @@ We have a frontend in React (Vite) and backend in Python.
 Please try to follow the guidelines.
 
 ### If you are looking to contribute to Backend (🐍 Python):
-- Check out our issues and contribute to `/application` or `/scripts` (ignore old `ingest_rst.py` `ingest_rst_sphinx.py` files; they will be deprecated soon).
+- Review our issues and contribute to /application or /scripts (please disregard old ingest_rst.py and ingest_rst_sphinx.py files; they will be phased out soon).
```
Comment (dartpain) [inline CONTRIBUTING.md:35]: Leave the word deprecated

**Review by dartpain — COMMENTED** [review]
Looks good, just left a few comments, thank you!

**Comment by HarshMN2345** [comment]
@dartpain done

File: CONTRIBUTING.md (hunk 1 above)
Comment (dartpain) [reply CONTRIBUTING.md]: Keep the: `We are all very grateful! `
Phrase

File: CONTRIBUTING.md (hunk 2 above)
Comment (dartpain) [reply CONTRIBUTING.md]: should be not bug, fixes or suggestions but `bugs, fixes or suggestions`

**Comment by dartpain** [comment]
Just two small issues

**Comment by HarshMN2345** [comment]
@dartpain done

**Review by dartpain — APPROVED** [review]

**Comment by dartpain** [comment]
@holopin-bot @HarshMN2345 Thank you!

**Pull request merged**